gbpservice/nfp/lifecycle_manager/networking/api.py

```diff
@@ -0,0 +1,41 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from gbpservice.nfp.lifecycle_manager.networking.drivers import (
+    gbp_driver)
+from gbpservice.nfp.lifecycle_manager.networking.drivers import (
+    neutron_driver)
+
+
+class NetworkingAPI(object):
```

Comment:
All the openstack_driver.NeutronClient() usage in dlc_manager, slc_manager should be abstracted by this class.